services/ingestion/app/main.py

```
@@ -0,0 +1,31 @@
+from services.ingestion.app.config import settings
+from services.ingestion.db.sqlite import init_db
+
+
+def setup_logging() -> None: 
+    logging.basicConfig(
+        level=settings.LOG_LEVEL,
+        format="%(asctime)s | %(levelname)s | %(name)s | %(message)s",
+    )
+    
+@asynccontextmanager
+async def lifespan(app: FastAPI):
```

Ruff (0.15.2)

[warning] Unused function argument: `app`

(ARG001)